index.html

```diff
@@ -62,5 +62,184 @@ <h1>Hi👋, I'm Cephas Tokoli</h1>
          </div>     
         </div>   
     </section>
+    <section class="grid">
+            <div>
+                <h2 class="headings">Projects</h2>
+            </div>
+     
+            <div class="project">
+                <div class="laptop-img">
+                    <img src="images/laptop.png" alt="laptop">
+                </div>
+                <div class="project-details">
+                    <div class="tech-details">
+                        <p class="project-name">Project name goes here</p>
```

Comment:
- [ ] I think it would be better to use `hx` tag for project name/title instead of `p` tag for better SEO 📌 What do you think? 💭